azure/template.json

```diff
@@ -253,38 +220,114 @@
                     }
                 }
             },
-            "dependsOn": []
+            "dependsOn": [
+            ]
         },
         {
-            "apiVersion": "2017-08-01",
-            "name": "worker-app-insights",
+            "condition": "[greater(length(parameters('apiCustomHostName')), 0)]",
+            "apiVersion": "2017-05-10",
+            "name": "api-app-service-certificate",
+            "resourceGroup": "[parameters('sharedBackEndAppServicePlanResourceGroupName')]",
             "type": "Microsoft.Resources/deployments",
             "properties": {
                 "mode": "Incremental",
                 "templateLink": {
-                    "uri": "[concat(variables('deploymentUrlBase'), 'application-insights.json')]",
+                    "uri": "[concat(variables('deploymentUrlBase'),'app-service-certificate.json')]",
                     "contentVersion": "1.0.0.0"
                 },
                 "parameters": {
-                    "appInsightsName": {
-                        "value": "[variables('workerAppServiceName')]"
+                    "keyVaultCertificateName": {
+                        "value": "[parameters('apiCertificateName')]"
                     },
-                    "attachedService": {
-                        "value": "[variables('workerAppServiceName')]"
+                    "keyVaultName": {
+                        "value": "[parameters('keyVaultName')]"
+                    },
+                    "keyVaultResourceGroup": {
+                        "value": "[parameters('keyVaultResourceGroup')]"
+                    }
+                }
```

Comment:
Can you use the serverFarmId parameter in here please

Reply:
Added in the serverFarmId parameter with latest commit 👍